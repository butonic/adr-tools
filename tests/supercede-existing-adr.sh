adr init docs madr
adr new First Record
adr new -s 2 Second Record
head -10 docs/0002-first-record.md
tail -5 docs/0003-second-record.md
