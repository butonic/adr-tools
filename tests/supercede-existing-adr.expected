adr init docs madr
docs/0001-use-markdown-architectural-decision-records.md
adr new First Record
docs/0002-first-record.md
adr new -s 2 Second Record
docs/0003-second-record.md
head -10 docs/0002-first-record.md
---
title: "2. First Record"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0002-first-record.md
---

* Status: superseded by [3. Second Record]({{< ref "0003-second-record.md" >}})
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12
tail -5 docs/0003-second-record.md
## Links <!-- optional -->

* [Link type] [Link to ADR] <!-- example: Refined by [ADR-0005](0005-example.md) -->
* â€¦ <!-- numbers of links can vary -->
* supersedes [2. First Record]({{< ref "0002-first-record.md" >}})
